static/js/components/search/CustomNoHits.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react';
+import { NoHits } from 'searchkit';
+
+export default class CustomNoHits extends NoHits {
+  render() {
+    if ((this.hasHits() || this.isInitialLoading() || this.isLoading()) && !this.getError()) {
+      return null;
+    }
+
+    if (this.getError()) {
+      const suggestion = this.getSuggestion();
+      const query = this.getQuery().getQueryString();
+      let infoKey = suggestion ? "NoHits.NoResultsFoundDidYouMean" : "NoHits.NoResultsFound";
+
+      return (
+        <div className="no-hits">
+          {this.translate(infoKey, {query:query, suggestion:suggestion})}
```

Comment:
@singingwolfboy empty query also appears when user tries to access ``/learners`` page. but he is not staff.
 thats why he dont see any results. i am going to show ``There are no results for this search.`` message for this case.